### ISSUE TITLE
[branch: phpcs-fixer] Missing DocBlock not detected

### ISSUE THREAD

**Issue by aik099** [squizlabs/PHP_CodeSniffer#243]
If it happens, that there is an inline comment before function declaration (which in turn belongs to another method), then it's treated as malformed DocBlock comment of that function:

``` php
/**
 * A simple function comment.
 *
 * @param string $str The string passed in by reference
 *
 * @return void
 * @return
 */
public function functionOutsideClass(&$str)
{
    return;
}//end functionOutsideClass()


function missingCommentOutsideClass()
{
    return;
}//end missingCommentOutsideClass()
```

Error at line with `missingCommentOutsideClass` declaration:
- in `phpcs-fixer` branch: `You must use "/**" style comments for a function comment (...Commenting.FunctionComment.WrongStyle)`
- in `master` branch: `Missing function doc comment (...Commenting.FunctionComment.Missing)`

Piece of code is taken from `FunctionCommentUnitTest.1.inc` test file of Squiz/PEAR FunctionComment sniff. It's line 166 there.
